File: node/8/SumOfPositive/sum_of_positive.js
function positiveSum(arr) {
    let sum = 0;
    for (let i = 0; i < arr.length; i++) {
        if (arr[i] >= 0) {
            sum += arr[i];
        }
    }
    return sum;
}

module.exports = positiveSum;

// console.log(positiveSum([1,2,3,4,5]));  //-> 15
// console.log(positiveSum([1,-2,3,4,5])); //-> 13
// console.log(positiveSum([])); //-> 0

File: Python/8/SumOfPositive/sum_of_positive.py
import sys


def positive_sum(arr):
    return sum([m for m in arr if m > 0])


if __name__ == "__main__":
    if len(sys.argv) == 1:
        nums = [int(n) for n in input('>>> Enter numbers to add with comma-separated: ').split()]
        print(positive_sum(arr=nums))
    else:
        sys.exit(1)
